tests/run-make/invalid-symlink-search-path/rmake.rs

```diff
@@ -0,0 +1,33 @@
+// In this test, the symlink created is invalid (valid relative to the root, but not
+// relatively to where it is located), and used to cause an internal
+// compiler error (ICE) when passed as a library search path. This was fixed in #26044,
+// and this test checks that the invalid symlink is instead simply ignored.
+// See https://github.com/rust-lang/rust/issues/26006
+
+//@ needs-symlink
+//Reason: symlink requires elevated permission in Windows
+
+use run_make_support::{create_symlink, rfs, rustc};
```

Comment:
Problem: yeah sorry, we no longer directly re-export `create_-_symlink` at the top level, it's under `rfs` now.